addons/intel/README.md

```diff
@@ -94,6 +94,10 @@ spec:
               - "-p"
               - "MON-HW"
             reportOutput: true
+          - args:
+              - "-p"
+              - "NAV-TIMELS"
+            reportOutput: true
```

Comment:
Should this be in the plugin code itself?

Reply:
I'm not sure - what are our pros/cons for putting ublox commands in ptpconfig Vs. the plugin code? On the Pros I thought that it's good to have an option to enable and mostly disable the command through the config. It comes useful in case users would want to apply the leap file manually or rely on a constellation other than GPS (that our code does not support ATM).
WDYT?

Reply:
we should have ability to disable dynamic leap update; to handle crisis